src/libraries/System.Runtime.Extensions/tests/System/Random.cs

```diff
@@ -477,13 +477,13 @@ public void Xoshiro_AlgorithmBehavesAsExpected()
                 Assert.Equal(9223372036854775803, randOuter.NextInt64(long.MaxValue - 5, long.MaxValue));
                 Assert.Equal(375288451405801266, randOuter.NextInt64(long.MinValue, long.MaxValue));
 
-                Assert.Equal(double.Parse("0.2885307561293763", CultureInfo.InvariantCulture), randOuter.NextDouble());
-                Assert.Equal(double.Parse("0.8319616593420064", CultureInfo.InvariantCulture), randOuter.NextDouble());
-                Assert.Equal(double.Parse("0.694751074593599", CultureInfo.InvariantCulture), randOuter.NextDouble());
+                Assert.Equal(0.2885307561293763, randOuter.NextDouble());
+                Assert.Equal(0.8319616593420064, randOuter.NextDouble());
+                Assert.Equal(0.694751074593599, randOuter.NextDouble());
 
-                Assert.Equal(float.Parse("0.7749006", CultureInfo.InvariantCulture), randOuter.NextSingle());
-                Assert.Equal(float.Parse("0.13424736", CultureInfo.InvariantCulture), randOuter.NextSingle());
-                Assert.Equal(float.Parse("0.05282557", CultureInfo.InvariantCulture), randOuter.NextSingle());
+                Assert.Equal(0.7749006, randOuter.NextSingle());
```

Comment:
These should end with `f` so it isn't implicitly converting and comparing against a double (not that it should matter in this case)